src/main.ts

```diff
@@ -7,8 +7,11 @@ import * as io from '@actions/io'
 import * as path from 'path'
 import * as fs from 'fs'
 
+const nightlyBuild =
+  (core.getInput('nightly-build') || 'false').toUpperCase() === 'TRUE'
 const baseUrl =
-  'https://download.tarantool.org/tarantool/release/' +
+  'https://download.tarantool.org/tarantool/' +
+  (nightlyBuild ? '' : 'release/') +
```

Comment:
There is explicit 'live' URL part, if you want.

In the next bullets 2 and 3 are equal, but the latter is more explicit.

1. https://download.tarantool.org/tarantool/release/<...>
2. https://download.tarantool.org/tarantool/live/<...>
3. https://download.tarantool.org/tarantool/<...>

Reply:
Thanks. Accepted.

Reply:
The trick didn't work:

```console
$ curl -v https://download.tarantool.org/tarantool/live/1.10/gpgkey

HTTP/1.1 302 Moved Temporarily
Location: https://www.tarantool.io/download/
```

I'll merge without it.

Reply:
That's the bug. Sorry. Okay.